pepper-apis/dss-core/src/test/resources/redisson-test-jcache.yaml

```diff
@@ -18,5 +18,5 @@ singleServerConfig:
   pingConnectionInterval: 3000
 threads: 16
 nettyThreads: 64
-codec: !<org.redisson.codec.FstCodec> {}
+codec: !<org.redisson.codec.Kryo5Codec> {}
```

Comment:
See [this comment in the other jcache file](https://github.com/broadinstitute/ddp-study-server/pull/2729/files#r1385528510).